src/ipc/ipc3/handler.c

```diff
@@ -299,6 +300,11 @@ static int ipc_stream_pcm_params(uint32_t stream)
 	reply.rhdr.error = 0;
 	reply.comp_id = pcm_params.comp_id;
 	reply.posn_offset = pcm_dev->cd->pipeline->posn_offset;
+
+	/* reset position value before send ipc */
+	memset(&posn, 0, sizeof(posn));
+	mailbox_stream_write(reply.posn_offset, &posn, sizeof(posn));
```

Comment:
Could you describe in which situations you see invalid position data being delivered to Linux?

Reply:
@lyakh yes,I saw this invalid value when I run alsa conformance test and I implement PCM pointer function to get host position value at the beginning. After I provide this code, the invalid value will be replaced with zero. Not sure if this issue could be reproducible on Intels platforms because it looks like Intel uses different ways to get position at Linux side. thanks.

Reply:
got it, thanks!